### PR TITLE
ADD Device Models. Device and DeviceModel

### PR DESCRIPTION
**Pull request by jmcanterafonseca**

### CODE REVIEW

**Comment by jmcanterafonseca** [comment]
@gtorodelvalle  r?

File: Device/Device/doc/spec.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
++ `softwareVersion` : The software version of this device.
+    + Attribute type: [Text](https://schema.org/Text)
+    + Optional
+
++ `firmwareVersion` : The firmware version of this device.
+    + Attribute type: [Text](https://schema.org/Text)
+    + Optional
+
++ `osVersion` : Thee version of the host operating system device.
```
Comment (gtorodelvalle) [inline Device/Device/doc/spec.md]: Thee -> The

File: Device/Device/doc/spec.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    + Attribute type: Reference to an entity of type [DeviceModel](../../DeviceModel/doc/spec.md).
+    + Optional
+
++ `batteryLevel` : Device's battery level. It must be equal to `1.0` when battery is full. `0.0` when battery ìs empty.
+`null` when cannot be determined. 
+    + Type: [Number](https://schema.org/Number)
+    + Allowed values: Interval [0,1]
+    + Attribute metadata:
+        + `timestamp`: Timestamp when the last update of the attribute happened.
+        This value can also appear as a FIWARE [TimeInstant](https://github.com/telefonicaid/iotagent-node-lib/blob/develop/README.md#TimeInstant)
```
Comment (gtorodelvalle) [inline Device/Device/doc/spec.md]: develop -> master

File: Device/Device/doc/spec.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+            + Type: [DateTime](http://schema.org/DateTime)
+    + Optional
+    
++ `deviceState` : State of this device from an operational point of view. Its value can be vendor dependent.  
+    + Type: [Text](https://schema.org/Text)
+    + Attribute metadata:
+        + `timestamp`: Timestamp when the last update of the attribute happened.
+        This value can also appear as a FIWARE [TimeInstant](https://github.com/telefonicaid/iotagent-node-lib/blob/develop/README.md#TimeInstant)
```
Comment (gtorodelvalle) [inline Device/Device/doc/spec.md]: develop -> master

File: Device/Device/doc/spec.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+    + Attribute type: [DateTime](https://schema.org/)
+    + Optional
+
++ `value` : A observed or reported value. For actuator devices, it is an attribute that allows
+a controlling application to change the actuation setting. For instance, a switch device which is currently *on* can report a value `"on"`of type `Text`.
+Obviously, in order to toggle the referred switch, this attribute value will have to be changed to `"off"`.
+    + Attribute type: Any type, depending on the device. Usually [Text](https://schema.org/Text) or [QuantitativeValue](https://schema.org/QuantitativeValue).
+    + Attribute metadata:
+        + `timestamp`: Timestamp when the last update of the attribute happened.
+        This value can also appear as a FIWARE [TimeInstant](https://github.com/telefonicaid/iotagent-node-lib/blob/develop/README.md#TimeInstant)
```
Comment (gtorodelvalle) [inline Device/Device/doc/spec.md]: develop -> master

**Pull request merged**